recipes/prometheus-async/recipe.yaml

```diff
@@ -0,0 +1,48 @@
+context:
+  name: prometheus-async
+  python_min: 3.7
```

Comment:
Conda-forge doesn't support <3.8 anymore. You should set this only if your min Python is higher than 3.9

```suggestion
```